### ISSUE TITLE
[input] error state color is not passing WCAG AA

### ISSUE THREAD

**Issue by ronyfinastra** [Finastra/finastra-design-system#449]
Hello UXG @ttalbot 
How are you?

There is a problem with the label color on the error state mode of the input field
which not pass the accessibility standard.

One of our costumers raised a flag about this matter
and i wonder if its possible to fix this issue in order to allow him to
upgrade to an improve accessible working environment?

the component pic
https://ibb.co/k5KL93b

the test result pic
https://ibb.co/Vxz7yxS

This is urgent
thanks
Rony

**Comment by ttalbot** [comment]
Hi Rony,

You're right, the input error state is not accessible enough.
Best way to solve this is to increase error color darkness a bit and increase lightness of error background until it passes AA.

It would be something like this:
#D60040 foreground
#F7EAED background

![image](https://user-images.githubusercontent.com/371041/140105430-2abe27d5-937a-42e5-bfba-0e3781792565.png)

**Comment by ttalbot** [comment]
Hi @ronyfinastra ,

We updated [the UI kit](https://www.figma.com/community/file/982657631921263317) with the new error color (#D60040).
The implementation is following shortly!

Thanks for helping us making the design system accessible 👍

**Comment by ronyfinastra** [comment]
@ttalbot 
Thank you Thomas
i apricate your efforts.

please ping me when the implementation will accomplish
the client's project manager is breathing down my neck :)

**Comment by ronyfinastra** [comment]
Thank you Thomas
